src/Transport/Http/HttpMethod.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * This file is part of the prooph/event-store-client.
+ * (c) 2018-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2018-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStoreClient\Transport\Http;
+
+/** @internal */
+class HttpMethod
+{
+    public const Get = 'GET';
```

Comment:
constant should be UPPERCASE not anymore?

Reply:
Why uppercase constant? It's hard to read and there is already a naming convention: CamelCase, why add another one?

Reply:
I violate PSR-2, no problem for me. It's a stupid rule. That everybody does it (me including in the past) is not an argument. Readability and consistency in naming convention is more important than stupid PSR-2 rules.

Reply:
> Readability and consistency in naming convention is more important than stupid PSR-2 rules

Yes, consistency, but in a much wider range than only one small project. PSR-2 Provides This Consistency over many many open source projects in the whole community. And I am glad this conventions exists. For me, it is more readable if every code I use is styled the same (or mostly the same). Every time I see code which does not stick to PSR-2 i´m distracted and stop reading. Thats already a big pain if constants are not written in UPPERCASE. Not because I like the rule to write it so. But I always would read my code the same way. 

You have plausible arguments for write to not use UPPERCASE, but there are also many arguments to write how it was defined in PSR-2. Thats the reason why this rules exists. We can stop to debate about code styles over and over again, since there almost always good args for booth sites. Some public PHP community people have decided it should be styled this way. In a democratic way. And now it is so much easier to maintain open source code, since all can hold on that rules without been distracted with endless discussions about how to do it. (tabs vs. whitespace)

Thats not only true for open source projects. I advocate this opinion also in consultations for customers and colleagues. Firstly, you safe yourself and all other employees endless design decisions. Secondly, it would be much easer for new colleagues to start reading and understand code they are not involved before. 

So, you can of course determine it for your self, if you would use any standard provided from the community. But you should consider to accept the community rules. This would be much easier as run in an endless discussion about design rules for your own project. Thats not worth the time, at all. 

By the way: I read constants like `GET` and `POST` not as words. More as symbols. Their meaning is clear at first glance, if you read them over and over again. Pretty much like traffic signs. But if one single person decide to write it like actually words, I have to read the whole word to understand here meaning. Thats a big distraction and the reason why 3 persons have found this issue in such a big PR.

Reply:
> Readability and consistency in naming convention is more important than stupid PSR-2 rules

In terms of consistency, is there not a Prooph standard? I see `UPPERCASE_CONSTANT` used elsewhere…

Reply:
You don't find any uppercase constants in this library here.